message_ix_buildings/chilled/util/util.py

```diff
@@ -9,6 +10,23 @@ def get_project_root() -> Path:
     return Path(__file__).parent.parent.parent
 
 
+def get_logger(__name__):
+    log = logging.getLogger(__name__)
+    log.setLevel(logging.INFO)
+
+    # configure the handler and formatter as needed
+    handler = logging.FileHandler(f"{__name__}.log", mode="w")
```

Comment:
```suggestion
def get_logger(name: str):
    log = logging.getLogger(name)
    log.setLevel(logging.INFO)

    # configure the handler and formatter as needed
    handler = logging.FileHandler(f"{name}.log", mode="w")
```
Probably best not to use dunder variables as your own function argument names. These variables have some magic about them that we probably don't want to disturb. 
Fun fact: if you want to indicate that a parameter should only be used internally, call it `_parameter`.